label_studio/tasks/validation.py

```diff
@@ -120,7 +120,7 @@ def check_allowed(task):
     @staticmethod
     def raise_if_wrong_class(task, key, class_def):
         if key in task and not isinstance(task[key], class_def):
-            raise ValidationError('Task[{key}] must be {class_def}'.format(key=key, class_def=class_def))
+            raise ValidationError('Task[{key}] must be {class_def.__name__}'.format(key=key, class_def=class_def))
```

Comment:
what if class_def = (list, dict) ? I think it will print 'tuple', that's incorrect

Reply:
Fixed.